tests/jsx_template/styled-components.js

```diff
@@ -0,0 +1,11 @@
+<style jsx>{`
+  p {
+    color: red;
+  }
+`}</style>;
+
+<style jsx>{tpl`
+  p {
+    color: red;
+  }
+`}</style>;
```

Comment:
Ahh, hmm... might be good to document that in a test here. 

Since existing versions of prettier transform into the whitespace version, users who've been using prettier won't get the preferable version.